README.md

```diff
@@ -370,3 +370,75 @@ column.
 
 Note, when using this expression. the thing you are counting is tags, not events, so if you
 have 10 events, each of which has 10 tags, then a `count()` of `tags_key` will return 100.
+
+
+##  Migrations
+
+Snuba provides functionality to manage ClickHouse database migrations. Migrations
+in Snuba are organized into groups, which typically correspond with features / sets
+of related tables in Snuba. In some cases, groups can provide a mechanism to test
+experimental features in Snuba that should not be rolled out to most users yet.
+Migration groups can be defined as either optional or mandatory. Optional groups
+can be toggled by the user via `settings.SKIPPED_MIGRATION_GROUPS`. In most cases,
+the default in settings.py should not be changed, and doing so may result in
+unexpected behavior.
+
+
+### Commands
+The `snuba migrations` group of commands provides some functions to manage migrations.
+
+#### List migrations
+`snuba migrations list`
+
+- Lists all migrations and their statuses
+
+#### Run all migrations
+`snuba migrations migrate --force`
+
+- Runs all pending migrations and brings your database to the latest state.
+- Running with the --force flag means that any migrations marked blocking (generally
+because they contain a data migration) will also be executed. Blocking migrations may
+take some time to complete. Running with --force assumes that any consumers filling
+the corresponding table are stopped and no new data is being written to the table
+as the migration is taking place.
+- If you are running `snuba devserver`, this command automatically run when the
+devserver is started and there is no need to manage migrations manually.
+
+#### Run a single migration
+`snuba migrations run --group <group> --migration-id <migration_id>`
+
+- Runs a single migration
+- Only allowed if there are no prior migrations in that group that have not been completed
+
+#### Reverse a single migration
+`snuba migrations reverse --group <group> --migration-id <migration_id>`
+
+- Reverses a single migration
+- Only allowed if there are no subsequent completed migrations in that group
+
+
+### Add a new migration
+In order to add a new migration, first determine which migration group the new
+migration should be added to, and add an entry to that group in `migrations/groups.py`
```

Comment:
It might be worth adding a few lines about how to create a new group as well.

Reply:
Done